syntax = "proto3";

package user;

service UserService {
    rpc GetUsers (Empty) returns (UserList);
}

message Empty {}

message User {
    int32 id = 1;
    string name = 2;
    string email = 3;
}

message UserList {
    repeated User users = 1;
}
